mobile/src/defaultViewComponents/defaultViewTable.js

```diff
@@ -68,22 +68,26 @@ export default class DefaultViewTable extends Component {
   }
 
   verifyData = (isBottomTable) => {
+    const sortByDate = (a,b) => b.dateCreate - a.dateCreate
     let items = Object.values(this.props.items)
     let newItems = items.filter(item => item.isResolved === false && item.isBlock !== true)
     if (this.props.currentFilter !== "All") {
       newItems = newItems.filter(item => item.type === this.props.currentFilter.toLowerCase())
       items = items.filter(item => item.type === this.props.currentFilter.toLowerCase())
     }
     if (isBottomTable) {
-      const liveItems = newItems.filter(item => item.creator.id !== client.currentUser.id)
-      liveItems = liveItems.sort((a,b) => b.dateCreate - a.dateCreate)
-      const resolvedItems = items.filter(item => item.isResolved)
-      resolvedItems = resolvedItems.sort((a,b) => b.dateCreate - a.dateCreate)
+      const liveItems = newItems
+      .filter(item => item.creator.id !== client.currentUser.id)
```

Comment:
it's a little easier to visualize the separation between statements with some indentation:

```javascript
const resolvedItems = items
  .filter(item => item.isResolved)
  .sort(sortByDate)
```